kucherov_sergeev/graph_printing.cpp

```diff
@@ -1,39 +1,47 @@
 #include "graph_printing.hpp"
 
-namespace {}
-
 namespace uni_course_cpp {
 namespace printing {
 namespace {
 std::vector<int> get_vertices_depth_distribution(const Graph& graph) {
   std::vector<int> vertices_depth_distribution = {};
   const auto depth = graph.get_depth();
 
-  for (Graph::Depth i = 0; i < depth; i++) {
+  for (Graph::Depth i = 0; i <= depth; i++) {
     vertices_depth_distribution.push_back(graph.get_depth_vertex_ids(i).size());
   }
 
   return vertices_depth_distribution;
 }
 
-std::unordered_map<Graph::Edge::Color, int> get_edges_color_distribution(
-    const Graph& graph) {
-  std::unordered_map<Graph::Edge::Color, int> edges_color_distribution{
-      {Graph::Edge::Color::Green, 0},
-      {Graph::Edge::Color::Red, 0},
-      {Graph::Edge::Color::Yellow, 0},
-      {Graph::Edge::Color::Grey, 0}};
+std::vector<int> get_edges_color_distribution(const Graph& graph) {
+  std::vector<int> edges_color_distribution(sizeof(Graph::Edge::Color), 0);
```

Comment:
Чем конвертировать одно в другое, лучше просто мап использовать
```suggestion
  std::map<Graph::Edge::Color, int> edges_color_distribution;
```